.eslintignore

```diff
@@ -0,0 +1,2 @@
+lib/
+node_modules/
```

Comment:
`node_modules` are ignored by default in `eslint`. 

https://eslint.org/docs/user-guide/configuring#eslintignore

```suggestion
```